spec/unit/helpers/check_helper_spec.rb

```diff
@@ -50,6 +50,43 @@
     expect { params[:test_param_three] }.to raise_error(/not defined in the specification/)
   end
 
+  context 'when created from stringified hash' do
+    let(:params) do
```

Comment:
Similar code found in check_helper_spec.rb:54, check_helper_spec.rb:60 (mass = 32)